metadata/io.etcd/jetcd-core/0.7.5/resource-config.json

```diff
@@ -0,0 +1,13 @@
+{
+  "resources": {
+    "includes": [
+      {
+        "condition": {
+          "typeReachable": "io.etcd.jetcd.launcher.EtcdContainer"
+        },
+        "pattern": "\\Qorg/testcontainers/utility/ResourceReaper.class\\E"
```

Comment:
Is this needed for the runtime of `jetcd-core` or just for unit tests?

Reply:
- This class directly refers to the class of testcontainers, so I think it is necessary, refer to https://github.com/etcd-io/jetcd/blob/jetcd-0.7.5/jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/EtcdContainer.java . In the javadoc this class is only used to start the Docker Container needed for the unit tests, do I need to mark something? Or move this entry to the test folder?

Reply:
I think you are right, I glanced over the sources and `EtcdContainer` seems to implement a class from `testcontainers` - I think the safest approach would be to keep it as is